docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,74 @@
+- What: Users can now easily restore their replica set on any other cluster with the same configuration. Please note that this feature is only available for replica sets and not for sharded clusters. Restoring sharded clusters is a complicated process that still requires manual intervention. For more information, please refer to our documentation.
+
+### Query Analytics - Meta information
+
+- Why: QAN is a useful tool for filtering data and identifying problematic queries. However, it can be difficult to determine which services, nodes, and other labels are associated with the identified queries.
+
+- What: This new feature assists the user in viewing all labels associated with the chosen query (metadata) to streamline the process of identifying and resolving issues.
+
+
+
+## Improvements
+
+- [PMM-6279](https://jira.percona.com/browse/PMM-6279) - [QAN]: Query metadata not visible in new UI
+- [PMM-10270](https://jira.percona.com/browse/PMM-10270) -[Backup Management]: Restore on a new replica set cluster
+- [PMM-11999](https://jira.percona.com/browse/PMM-11999) - [Advisors]: Expose check family field in advisors API
+- [PMM-12257](https://jira.percona.com/browse/PMM-12257) - [Advisors]: The **Advisors** page now includes a new **Technology** column, showing the database type for each advisor in the table. This helps you easily differentiate between the MongoDB, MySQL and PostgreSQL	advisors.
+- [PMM-11637](https://jira.percona.com/browse/PMM-11637) - In PMM 2.39.0, we updated the interface to reflect the new PMM logo as per the new branding.
+- [PMM-11206](https://jira.percona.com/browse/PMM-11206) - [Alerting]: Enhanced the **Add rule template** dialog box to support YAML content for multiple alert templates. Alert templates added in bulk are displayed individually on the **Alert rule templates** page.
+
+ 
+## Bugs Fixed
+
+- [PMM-12151](https://jira.percona.com/browse/PMM-12151) - [QAN]: Explain plan not displayed if database name is missing in the query
+- [PMM-10391](https://jira.percona.com/browse/PMM-10391) - PMM did not work with external clickhouse versions greater than 22.5.2. The issue has been resolved now.
+- [PMM-11658](https://jira.percona.com/browse/PMM-11658) - pmm-agent crashes simultaneously on all nodes
+- [PMM-11992](https://jira.percona.com/browse/PMM-11992) - Database name is ignored while adding any Postgres
+- [PMM-12161](https://jira.percona.com/browse/PMM-12161) - "MongoDB Cluster Summary" - too many "Config Services"
+- [PMM-12231](https://jira.percona.com/browse/PMM-12231) - PMM disallows to install, update and remove plugins
+- [PMM-12263](https://jira.percona.com/browse/PMM-12263) - Wrong Replication Lag by Set values if RS is PSA - MongoDB Cluster Summary
```

Comment:
This one seems to be too cryptic, I suggest we ask the author of the bugfix to provide a better title.

Reply:
+1 to @atymchuk . we need to explain this

Reply:
@rnovikovP @atymchuk I have not understood this from the ticket. Could you please help me with the explanation.

Reply:
On the [MongoDB Cluster Summary](https://pmmdemo.percona.com/graph/d/mongodb-cluster-summary/mongodb-cluster-summary?orgId=1&refresh=1m&search=open) dashboard, the Replication Lag by Set graph showed the wrong value if rs was in the primary-secondary-arbiter (PSA) configuration. It calculated the arbiter's lag from the beginning of the UNIX time (currently >54 years instead of a few seconds). The issue has been resolved now.

@rnovikovP Please confirm if this is correct?